schema_validation.py

```
@@ -0,0 +1,279 @@
+        print(f"Error removing test device {mac}: {e}")
+    finally:
+        if cur:
+            cur.close()
+        if conn:
+            conn.close()
+
+
+def inject_test_device():
+    # Use locally administered MAC address range
+    mac = "02:%02x:%02x:%02x:%02x:%02x" % (
+        random.randint(0, 255), random.randint(0, 255), random.randint(0, 255),
```

Ruff (0.14.11)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)